wpeverest/user-registration
# Fix - Form not submitting issue while captcha keys are not set.

### All Submissions:






### Changes proposed in this Pull Request:

Previously, there was a form not submitting issue while CAPTCHA keys are not set. This PR fixes this issue.

### How to test the changes in this Pull Request:

1. Go to Settings > Integration > CAPTCHA and set the captcha keys empty (try for all captcha versions).
2. Try to submit the form and login form also(in both cases ajax and post-submission).
3. And Expected result: there should be no captcha error.
4. Also, test all captcha versions by setting proper captcha keys.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Form not submitting issue while captcha keys are not set.
